source/managing-teams/coaching-reports.html.md

```diff
@@ -0,0 +1,130 @@
+tackling resources that will close that gap. As you go through the resources and get closer to your
+vision, you continuously re-evaluate and re-define the vision.
+
+Learning paths can contain pretty much any resource that will help close the gap between vision and
+reality: it can be a book, a conference talk, an article, or a task. No matter the resource, you
+should encourage the teammate to take some time after tackling the resource to reflect on and
+discuss what they just read/watched/did. Grappling with ideas is the best way to solidify them.
+
+When designing a learning path with your report, try to keep it realistic and don't overdo it. It
+can be tempting to fill the path with different topics and resources, but this can get overwhelming
+very quickly. Instead, remember that there's always time to expand the learning path, and tackle it
+one chunk at a time.
+
+## Creating space for learning
+
+Learning and growth at Nebulab mainly happen in two environments: while working on engagements and
+during [Investment Time](/personal-growth/investment-time/).
+
+Being a consulting firm means we don't have complete control over our day-to-day work environment.
+This, in turn, means that learning on client engagements must happen in a controlled, safe manner
+that will not hurt performance. You know your client best, so you should figure out a
+setup that will allow the team to learn while also delivering on business initiatives. In our
+experience, this usually means monitoring "experiments" closely to ensure that they stay within safe
+boundaries, and that key outcomes are achieved within the established timelines.
+
+When it comes to Investment Time, you and your report have complete freedom over how to use it.
+As a manager, keep in mind that Investment Time should be a benefit, not a chore: you shouldn't
+force someone to use it for learning, if they don't seem interested in doing so.  
+
+## Providing feedback
+
+Finally, a key aspect of coaching reports is providing feedback about their performance and growth.
+Direct feedback is the best way to coach your reports, and it should be a regular part of your day
+to reflect on their performance and think about what positive or constructive feedback you can
+provide.
+
+At Nebulab, there are multiple rituals that involve providing feedback to your report. Each ritual
+has a different cadence and a different "scope of impact", and you should use all of them in a
+holistic and strategic way:
+
+- **Situational feedback** is feedback about a specific event. For example, you might your report
+  did a particularly good job of handling a project, or a particularly poor job of communicating
+  with a stakeholder. In this case, you should provide the feedback immediately, so that you can
+  have a relevant and useful conversation while the memory is still fresh in everyone's mind.
+- **[One-on-ones](/managing-teams/holding-one-on-ones/)** are best reserved for more elaborate
```

Comment:
Can we uniform how we refer to `1-1s`? Sometime it's `One-on-ones`, what's the difference?